R/forcats-package.r

```diff
@@ -0,0 +1,10 @@
+#' forcats.
+#'
+#' Helpers for reordering factor levels (including moving specified levels to
+#' front, ordering by first appearance, reversing, and randomly shuffling), and
+#' tools for modifying factor levels (including collapsing rare levels into
+#' other, anonymising, and manually recoding).
+#'
+#' @name forcats
+#' @docType package
+NULL
```

Comment:
I think this presumes the dev version of roxygen2; I was using CRAN. You still want?